ddtrace/internal/telemetry/writer.py

```diff
@@ -428,6 +426,10 @@ def _app_integrations_changed_event(self, integrations):
         }
         self.add_event(payload, "app-integrations-change")
 
+    def _app_dependencies_loaded(self):
```

Comment:
I don't think this function name conveys the fact that the function adds an event to `self`.